pkg/cluster/internal/loadbalancer/const.go

```diff
@@ -20,7 +20,7 @@ package loadbalancer
 const ControlPlanePort = 6443
 
 // Image defines the loadbalancer image:tag
-const Image = "nginx:1.15.12-alpine"
+const Image = "haproxy:1.9.8-alpine"
```

Comment:
we should go back to the previous approach of using our own image with a default config. I have this WIP.